st2common/st2common/openapi.yaml

```diff
@@ -3750,15 +3750,15 @@ definitions:
       status:
         description: The current status of the action execution.
         type: string
-        enum: {{ LIVEACTION_STATUSES }}
```

Comment:
To confirm - this file is automatically generated based on .j2 definition on make target and build?